contributing.md

```diff
@@ -1,96 +1,88 @@
 
-This project adheres to the Contributor Covenant [code of conduct](code-of-conduct.md). By participating, you are expected to uphold this code. Please report unacceptable behavior to hello@okteto.com.
+This project adheres to the Contributor Covenant [Code of Conduct](CODE_OF_CONDUCT.md). By participating, you are expected to uphold this code. Please report any unacceptable behavior to hello@okteto.com.
+
+## Ways To Contribute
+
+### Reporting Issues
 
-## File an Issue
+Reporting issues is a great way to help the project! This isn't just limited to reporting bugs but can also include feature requests or suggestions for improvements in current behavior. We use [GitHub issues](https://github.com/okteto/okteto/issues) for tracking all such things. But if you want to report a sensitive security issue or a security exploit, you can directly contact the project maintainers on hello@okteto.com or via [a Twitter DM](https://twitter.com/oktetoHQ).
 
-Not ready to contribute code, but see something that needs work? While we encourage everyone to contribute code, it is also appreciated when someone reports an issue. We use [GitHub issues](https://github.com/okteto/okteto/issues) for this.
+### Contributing Code
 
-## Report security issues
+When contributing features or bug fixes to Okteto CLI, it'll be helpful to keep the following things in mind:
 
-If you want to report a sensitive security issue, or a security exploit, you can directly contact the project maintainers via [Twitter DM](https://twitter.com/oktetoHQ) or via hello@okteto.com.
+- Communicating your changes before you start working
+- Including unit tests whenever relevant
+- Making sure your code passes the [lint checks](#lint)
+- Signing off on all your git commits by running `git commit -s`
+- Documenting all Go public types, structs, and functions in your code
 
-## Pull Requests
+Discussing your changes with the maintainers before implementation is one of the most important steps, as this sets you in the right direction before you begin. The best way to communicate this is through a detailed GitHub issue. Another way to discuss changes with maintainers is using the [#okteto](https://kubernetes.slack.com/messages/CM1QMQGS0/) channel on the Kubernetes slack.
 
-When submitting a pull request, please make sure that it adheres to the following standard.
+#### Making a Pull Request
 
-1. Code should be go fmt compliant.
-1. Public types, structs and funcs should be documented.
-1. It includes pertinent unit tests.
-1. Commits are signed-off (`git commit --signoff or -s`).
-1. It includes the whole template for issues and pull requests.
-1. It [references addressed issues](https://help.github.com/en/github/managing-your-work-on-github/closing-issues-using-keywords) in the PR description & commit messages.
-1. It has clear commit messages.
+The following steps will walk you through the process of opening your first pull request:
 
-## Sign your work
+##### Create a Clone of Your Fork
```

Comment:
this reads odd to me. maybe just go with 'Fork the repository'?

Reply:
Changed!